addons/discuss-button/compact-nav.css

```diff
@@ -13,8 +13,8 @@
   display: block;
   font-size: 0.75rem;
   font-weight: normal;
-  line-height: normal;
-  padding-bottom: 8px;
+  line-height: 0;
```

Comment:
This used to be 0 but was changed in #5716 to prevent lines from overlapping when the username wraps:
![image](https://github.com/user-attachments/assets/3ca87ff4-cf3f-4b52-a18b-bd1a5bfe6d8d)